.gitignore

```diff
@@ -132,7 +132,7 @@ dist
 
 # Others
 **/.DS_Store
-
+.dccache
```

Comment:
[Q] Why is this needed ?

Reply:
I think it was created by snyk scanner, so thought of keeping it as people might run it locally and accidentally submit it to the repo